README.md

```diff
@@ -52,13 +52,10 @@ See [install.md](docs/install.md) for a detailed tutorial.
 
 Security
 ---
-* Nix package manager, NixOS and packages can be built from source to reduce reliance on binary caches.
-* Builds happen in a [sandboxed environment](https://nixos.org/nix/manual/).
-* Packages dependencies are [pinned](pkgs/nixpkgs-pinned.nix). Most packages are built from the [nixos stable channel](https://github.com/NixOS/nixpkgs-channels/tree/nixos-19.03), with a few exceptions that are built from the nixpkgs unstable channel.
-* nix-bitcoin merge commits are signed.
-* nix-bitcoin is built with a [hardened kernel](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/profiles/hardened.nix) by default.
-* Services operate with least privileges. They each have their own user and are restricted further with [systemd options](modules/nix-bitcoin-services.nix).
-* There's a non-root user *operator* to interact with the various services.
+* **Simplicity:** Only services you select in `configuration.nix` and their dependencies are installed, packages and dependencies are [pinned](pkgs/nixpkgs-pinned.nix), most packages are built from the [nixos stable channel](https://github.com/NixOS/nixpkgs-channels/tree/nixos-19.03), with a few exceptions that are built from the nixpkgs unstable channel, builds happen in a [sandboxed environment](https://nixos.org/nix/manual/), code is continiously reviewed and refined.
+* **Integrity:** Nix package manager, NixOS and packages can be built from source to reduce reliance on binary caches, nix-bitcoin merge commits are signed, all commits are approved by multiple nix-bitcoin developers with a strong focus on security, upstream packages are cryptographically verified where possible, we use this software ourselves.
```

Comment:
perhaps "with a strong focus on security" is a bit too much

Reply:
Not that it's wrong... but as it stands sounds very much like marketing because it's void of actual information.